src/Controller/UserController.php

```diff
@@ -5,9 +5,87 @@
 
 namespace App\Controller;
 
+use App\Entity\User;
+use App\Form\user\EditUserFormType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
 
+/**
+ * Class UserController
+ * @package App\Controller
+ *
+ * @IsGranted("ROLE_ADMIN", message="Vous n'avez pas les droits d'accès.")
+ */
 class UserController extends AbstractController
 {
+    /**
+     * @return array
+     *
+     * @Route("/userlist")
+     *
+     * @Template("user/list.html.twig")
+     */
+    public function list()
+    {
+        $users = $this->getDoctrine()
+            ->getRepository(User::class)
+            ->findAll();
+
+        return array('users' => $users);
```

Comment:
return ['users' => $this->getDoctrine()
            ->getRepository(User::class)
            ->findAll()]
;